### PR TITLE
{data}[intel/2019a] mkl-service v2.0.1 w/ Python 2.7.15 + 3.7.2

### PR DESCRIPTION
**Pull request by boegel**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node2023.delcatty.os - Linux centos linux 7.6.1810, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/b47fe3894a6878dd1ad718a90ebde5f3 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node3100.skitty.os - Linux centos linux 7.6.1810, Intel(R) Xeon(R) Gold 6140 CPU @ 2.30GHz, Python 2.7.5
See https://gist.github.com/d728ad293baff9b2f2b081580239b91a for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node2594.golett.os - Linux centos linux 7.6.1810, Intel(R) Xeon(R) CPU E5-2680 v3 @ 2.50GHz, Python 2.7.5
See https://gist.github.com/dcc9a40202a6c6eb2f68d20713342dd9 for a full test report.

File: easybuild/easyconfigs/m/mkl-service/mkl-service-2.0.1-intel-2019a.eb [hunk 1]
```diff
@@ -0,0 +1,33 @@
+easyblock = 'PythonPackage'
+
+name = 'mkl-service'
+version = '2.0.1'
```
Comment (smoors) [inline easybuild/easyconfigs/m/mkl-service/mkl-service-2.0.1-intel-2019a.eb]: (note that version 2.0.2 is out)

File: easybuild/easyconfigs/m/mkl-service/mkl-service-2.0.1-intel-2019a.eb [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+
+homepage = 'https://github.com/pydata/xarray'
+description = """xarray (formerly xray) is an open source project and Python package that aims to bring
+ the labeled data power of pandas to the physical sciences, by providing N-dimensional variants of the
+ core pandas data structures."""
```
[the remark this answers is not in the document]
Comment (boegel) [reply easybuild/easyconfigs/m/mkl-service/mkl-service-2.0.1-intel-2019a.eb]: oops, fixed in 3580427fcc

File: easybuild/easyconfigs/m/mkl-service/mkl-service-2.0.1-intel-2019a.eb (hunk 1 above)
Comment (boegel) [reply easybuild/easyconfigs/m/mkl-service/mkl-service-2.0.1-intel-2019a.eb]: Yeah, might as well update... done in 3580427fcc

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node3100.skitty.os - Linux centos linux 7.6.1810, Intel(R) Xeon(R) Gold 6140 CPU @ 2.30GHz, Python 2.7.5
See https://gist.github.com/263152005befe08a58b69cb8b57e8d66 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node2023.delcatty.os - Linux centos linux 7.6.1810, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/c4f3bc4641997d45e1aa4f87c5b67e89 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node2420.golett.os - Linux centos linux 7.6.1810, Intel(R) Xeon(R) CPU E5-2680 v3 @ 2.50GHz, Python 2.7.5
See https://gist.github.com/1fbabccee7256d2003e4036c40e412a5 for a full test report.

**Comment by smoors** [comment]
Test report by @smoors
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
nic401 - Linux centos linux 7.5.1804, Intel(R) Xeon(R) Gold 6148 CPU @ 2.40GHz, Python 2.7.5
See https://gist.github.com/041d448bbf0f8d131a8c39ab2aa9519d for a full test report.

**Comment by smoors** [comment]
Going in, thanks @boegel!

**Pull request merged**